torch/csrc/jit/tensorexpr/expr.h

```diff
@@ -295,6 +302,50 @@ class TORCH_API Buf : public ExprNode<Buf> {
     return true;
   }
 
+  bool is_contigous(
+      at::MemoryFormat memory_format = at::MemoryFormat::Contiguous) {
```

Comment:
I think we can mark this function as `const` and also move its implementation to `expr.cpp`.